tianshou/utils/net/common.py

```diff
@@ -9,46 +9,134 @@
 def miniblock(
     inp: int,
     oup: int,
-    norm_layer: Optional[Callable[[int], nn.modules.Module]],
+    norm_layer: Optional[Callable[[int], nn.modules.Module]] = None,
+    activation: Optional[nn.modules.Module] = nn.ReLU,
 ) -> List[nn.modules.Module]:
     """Construct a miniblock with given input/output-size and norm layer."""
     ret: List[nn.modules.Module] = [nn.Linear(inp, oup)]
     if norm_layer is not None:
         ret += [norm_layer(oup)]
-    ret += [nn.ReLU(inplace=True)]
+    ret += [activation(inplace=True)]
     return ret
 
 
-class Net(nn.Module):
+class MLP(nn.Module):
     """Simple MLP backbone.
+    Create a MLP of size inp_shape*hidden_layer_size[0]*hidden_layer_size[1]...
+
+    :param hidden_layer_size: shape of MLP passed in as a list, not incluing
```

Comment:
'hidden_layers_sizes' instead 'hidden_layer_size'

Reply:
I still suggest using hidden_layers_size for consistency with history.

Reply:
> I still suggest using hidden_layers_size for consistency with history.

I don't think so. We should break consistency at this point, otherwise it may be confusing.

How about `hidden_size` or some other names?

Reply:
> I don't think so. We should break consistency at this point, otherwise it may be confusing (since we change API, an explicit warning should be generated when users use the network with old fashion)

exactly. `hidden_sizes` since it is a list, not a scalar.